tests/test_expressions.py

```diff
@@ -350,6 +350,7 @@ def test_function_building(self):
         )
 
         self.assertIsInstance(exp.func("instr", "x", "b", dialect="mysql"), exp.StrPosition)
+        self.assertIsInstance(exp.func("instr", "x", "b", dialect="sqlite"), exp.StrPosition)
```

Comment:
This isn't enough to test this, you need to do `self.validate_identity("INSTR(x, y)")` within `test_sqlite.TestSQLite.test_sqlite`.